db/schema.sql

```diff
@@ -277,7 +277,6 @@ CREATE TABLE emailed_anomaly (
 	CONSTRAINT PRIMARY KEY (id),
 	CONSTRAINT foreign_aws_account FOREIGN KEY (aws_account_id) REFERENCES aws_account(id) ON DELETE CASCADE
 );
-
```

Comment:
There many with and many withtout empty line, what should we do everywhere ?

Reply:
I don't think we really car about that, but I added the line back